src/index.js

```diff
@@ -1,12 +1,23 @@
 import BitfinexPriceFeeds from './bitfinex-price-feed.js'
 import fs from 'fs'
+import Oracle from './oracle.js'
+
+const PRIVATE_KEY = process.env.ORACLE_PRIVATE_KEY
+if (!PRIVATE_KEY) {
+  console.error('Missing ORACLE_PRIVATE_KEY env var');
+  process.exit(1);
+}
```

Comment:
We would like to be able to run it in both modes thus it should be optional parameter.
This might also require a change either here https://github.com/synonymdev/slashtags-widget-price-feed/pull/8/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R17 or in oracle constructor as well as here https://github.com/synonymdev/slashtags-widget-price-feed/pull/8/files#diff-240b28eeb8cb217a4219334c7a6db3404eae7fbb0397a55fb3f5eee6e08094b0R103 to account for optional oracle